### ISSUE TITLE
CC0

### ISSUE THREAD

**Issue by tbeu** [nemequ/portable-snippets#28]
According to https://opensource.org/faq#cc-zero, the OSI does not recommend to release OSS as public domain or CC0. Do you see any chance to change the license to BSD style, MIT or ISC?

**Comment by nemequ** [comment]
No.  I used to use MIT for everything, and regularly got requests to change to public domain, other licenses, or just complaints about MIT.  Ditto with LGPL.  Since switching to CC0 I almost never hear complaints but do get a lot of people expressing appreciation; it seems people like CC0 a lot more.

I'm aware of the problem with public domain in some jurisdictions, which is why I went with CC0 instead of an SQLite-like declaration.  Based on the link you sent it seems like what people objected to is how the fallback license explicitly mentions that it is not a patent grant.  AFAIK none of the other licenses you mentioned include a patent grant either; just because there is no clause explicitly withholding patents doesn't mean there is a grant, I just don't see a problem here. Frankly, I think explicitly mentioning that it is not a patent grant is better since it helps make people aware of the issue instead of just sweeping it under the rug.  Since it is not in my power to grant any patent licenses for anything in portable-snippets (because I don't own any) a patent grant, or simply omitting their mention, seems disingenuous.

What I might be willing to do is add a declaration that says that in the event the fallback is triggered (*i.e.,*, the public domain dedication is deemed invalid) the user is free to use the code under the terms of the MIT license instead of the fallback license in the CC0.  If you (or anyone) would like to suggest some text for that I'd be happy to review it.
